web/bfex/components/data_pipeline/tasks/scrape.py

```diff
@@ -3,31 +3,32 @@
 from bfex.models import Faculty
 from bfex.common.utils import URLs, FacultyNames
 from bfex.components.data_pipeline.tasks.task import Task
-
+from bfex.components.key_generation.rake_approach import *
+from bfex.components.data_pipeline.tasks.keywords import UpdateKeywordsFromScrape
```

Comment:
If you add keywords to the __init__.py of bfex/components/data_pipeline/tasks, it makes the imports a little nicer, but thats not necessary.